### DISCUSSION TITLE
等一手semi pro

### DISCUSSION THREAD

**Discussion by DroganC** [DouyinFE/semi-design#195] [Q&A] [answered]

**Comment by anymost** [comment]
Bug Label is weired

**Comment by zjz1993** [comment]
在来一手semi-design-mobile

**Comment by lovesL** [comment]
期待semi pro

**Comment by tank0317** [comment, marked as answer]
Semi 体系内的Pro的定位与其他组件库提供的 XX Pro不太一样，其他组件库的 xx Pro一般为脚手架+页面模板（表单页、表格页、etc）集合，这类产品我们在内部之前已经有过较长时间的实践验证，没有取得太明显的收益以及复用价值。
因此我们并不计划单独提供这种类型的模板（实际上这种模板也仅仅是一些组件的使用Demo集合，社区也可以自行拼装）。

我们规划的Pro会更多地与上游设计工具进行打通，目前还在内部打磨。后续有明确的milestone后会再开放给社区。

**Reply by hinego** [reply]
旧事重提 希望能提供一个简单但完善的后台布局（实现左边菜单 和上面的nav 同时适配手机显示）

这样像我这种不太会前端的也可以依靠这种框架做出不错的后台。。。。

**Reply by yuanzhixiang** [reply]
即将满一年，还没看到打磨出的产品

**Reply by lovesL** [reply]
哎 一年 一年 又一年 得兑现呀

**Reply by Axiano** [reply]
还在等待

**Reply by l0g1n** [reply]
2025了，希望提供一个

**Comment by xmsz-stu** [comment]
搞不懂pro的意义是什么？
首先第一层是基础component，下一层是block，再下一层是template

component不用说，就是拿来复用的
block呢？大部分情况也是可以复用的，比如一些表单、登录框等，直接复用省心省力
template呢？实际能复用的太少了...  因为一般都要定制，定制了template就没啥用了，而且template维护很麻烦，没啥效益


任何这里就要区分一下

国外也有pro，template。但是有第一大前提是收费，那这样才能去维护，而且质量比较高。加上定制性需求少(老板和用户没那么多莫名其妙的习惯)，所以也是能用。

国内的pro，好像只是要一个完整的后台模板而已，简单说就只是要个壳，实际上自己也就10分钟...
